src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/CollectDumpActionTests.cs

```diff
@@ -24,9 +24,8 @@ namespace Microsoft.Diagnostics.Monitoring.Tool.UnitTests
 {
     public sealed class CollectDumpActionTests
     {
-        private const string TempEgressDirectory = "/tmp";
         private const string ExpectedEgressProvider = "TmpEgressProvider";
-        private const string DefaultRuleName = "Default";
+        private const string DefaultRuleName = "DumpTestRule";
```

Comment:
Sorry, I had noticed it was out of date compared to what we had done with GC Dump so I had adjusted them in here since I figured it didn't need its own PR. That being said, I can pull these changes out so they go in their own PR. Yes, I just gave it a more descriptive Default Rule Name and switched to using TemporaryDirectory